daemon/algod/api/algod.oas2.json

```diff
@@ -3900,6 +3969,93 @@
           }
         }
       }
+    },
+    "SimulationUnnamedGroupResources": {
+      "description": "If unnamed resource access is allowed, this is the subset of unnamed resources that were accessed by this group and benefit from group resource sharing. Also see property 'unnamed-resources' in SimulateTransactionResult to see unnamed resources accessed by each transaction which cannot benefit from group sharing.",
```

Comment:
Can you add a little more around the word "benefit"? In my head it means, these unnamed resources were also each accessed by another transaction in the group.

Reply:
This is meant to indicate that if resources are present in this object, you can place them anywhere in the group. The txns that need to access them can do so with resource sharing.

Reply:
Am I understanding that these are things that are accessed in v9 programs?  So maybe it should read "and *could* benefit".

Are you going to separately report resources that must appear in certain txns because they are accessed by pre v9 programs?  I guess I'll find out as I read. Sounds complex.

Reply:
Good point, I've changed the wording to this in f343aa459:

> If unnamed resource access is allowed, this is the subset of unnamed resources that were accessed by this group and could benefit from group resource sharing. In other words, there are no restrictions on where these resources must be placed in the transaction group. Also see property 'unnamed-resources-accessed' in SimulateTransactionResult to see unnamed resources accessed by each transaction which cannot benefit from group sharing; in contrast, these resources must be placed local to that transaction.